site/src/sphinx/setup-configuration.rst

```diff
@@ -200,9 +221,14 @@ Once you have an access to a ZooKeeper cluster, update the ``replication`` secti
             "host": "*",
             "port": 36462
           },
-          "protocol": "http"
+          "protocol": "https"
```

Comment:
Same here. Let's add a dedicated section for TLS configuration and keep using `"protool": "http"` and `"tls": null`